README.md

```diff
@@ -139,6 +139,7 @@ Table of Contents
 
 ## APIs, Data and ML
 
+  * [Firecamp](https://firecamp.app/) — API studio to test out any APIs including Rest, Websocket, GraphQL and manymore. 
```

Comment:
really nice unfortunatly though this is not a service really is it? Isnt it just a standalone app?

Reply:
@ripienaar then I'll update you in future if changes happened :)